test/e2e/framework/service/const.go

```diff
@@ -36,10 +36,6 @@ const (
 	// liberal. Fix tracked in #20567.
 	KubeProxyLagTimeout = 5 * time.Minute
 
-	// KubeProxyEndpointLagTimeout is the maximum time a kube-proxy daemon on a node is allowed
-	// to not notice an Endpoint update.
-	KubeProxyEndpointLagTimeout = 30 * time.Second
```

Comment:
unrelated to the PR, do you measure `endpointLagTimeout` in scalability?

Reply:
We do - the exact definition is here:
https://github.com/kubernetes/community/blob/master/sig-scalability/slos/network_programming_latency.md